Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.6-alpine
```

Comment:
This is going to conflict with master, where we've added a new `calico/cni` docker image for shipping CNI binaries.